### ISSUE TITLE
topic naming convention

### ISSUE THREAD

**Issue by lazafi** [srothh/ppmrob#10]
I suggest following naming for topics:

/{publisher_node}/{topic_name}

for the "tello" node I would substitute "drone"

eg:
/drone/camera
/drone/telemetry

**Comment by VrabelPeter** [comment]
I agree, this also adheres to the [convention](https://wiki.ros.org/ROS/Patterns/Conventions).

**Comment by srothh** [comment]
Looks good!
